src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -1289,11 +1370,82 @@ int RadosBucket::commit_logging_object(const std::string& obj_name,
   }
 
   const auto temp_obj_name = to_temp_object_name(this, obj_name);
+
+  if (async) {
+    auto ioctx_logging = store->getRados()->get_logging_pool_ctx();
+    std::string target_obj_name = get_tenant() + "/" + get_name() + "/" + get_bucket_id() + "/" + prefix;
+    // Add entry to the global log object commit list
+    {
+      std::string global_list_name = "bucket_logging_commit_list";
+      bufferlist empty_bl;
+      std::map<std::string, bufferlist> new_commit_list{{target_obj_name, empty_bl}};
+      librados::ObjectWriteOperation op;
+      op.create(false);
```

Comment:
maybe change to `create(true)` an then you dont need to address the `-ENOENT` case?